.coderabbit.yml

```
@@ -0,0 +1,15 @@
+  high_level_summary: true
+  poem: false
+  review_status: false
+  collapse_walkthrough: true
+
+# Enable auto review for non-draft PRs
+auto_review:
+  enabled: true
+  drafts: false
+  ignore_title_keywords:
+    - "WIP"
+    - "DRAFT"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)